.github/workflows/techlabblog-deploy-dev.yml

```diff
@@ -0,0 +1,57 @@
+name: Techlab Blog | Deploy Dev
+
+on:
+  push:
+    branches: [ft/techlabblog-workflow]
```

Comment:
@kelvinkipruto I think this needs to be main
```suggestion
    branches: [main]
```

Reply:
Can you at least comment why you've marked it as resolved without addressing it?

Reply:
I was testing and forgot about this